src/config/dynamic-theme-fixes.config

```diff
@@ -22904,6 +22904,18 @@ body {
 
 ================================
 
+simracingmachines.com
+
+INVERT
+li[id^="Slide-multicolumn-"] img[class="multicolumn-card__image"][src*="logo"]
```

Comment:
Can you add one more invert for partners? They are black on dark too.
`li[id^="Slide-template-"] img[class="multicolumn-card__image"]`

I just don't want to mess with your pull request.

Reply:
I mean this part: (screen with my proposed invert added)
![obraz](https://github.com/darkreader/darkreader/assets/56877029/6a5bc455-8179-4c7a-990a-568a4f185271)

Reply:
@Myshor Good catch, I didn't see that.

I've updated the MR.